pkg/webhook/server/server.go

```diff
@@ -83,3 +92,25 @@ func GetWebhookServer() *webhook.Server {
 func IsSetUp() bool {
 	return webhookServer != nil
 }
+
+func findTlsCrtAndKey() (key, crt string, err error) {
+	// Mounted from service serving-CA secret: tls.key and tls.crt
+	if _, err := os.Stat(filepath.Join(webhookServerCertDir, "tls.key")); err == nil {
+		key = "tls.key"
+		if _, err := os.Stat(filepath.Join(webhookServerCertDir, "tls.crt")); err != nil {
+			return "", "", fmt.Errorf("could not find certificates for webhook: could not find tls.crt")
+		}
+		crt = "tls.crt"
+		return key, crt, nil
+	}
+	// Mounted by OLM: apiserver.key and apiserver.crt
+	if _, err := os.Stat(filepath.Join(webhookServerCertDir, "apiserver.key")); err == nil {
```

Comment:
When I install WebTerminalOperator with `make olm_install_local` I get 
![Screenshot_20200703_135935](https://user-images.githubusercontent.com/5887312/86463276-d9032600-bd35-11ea-9c69-2905710002d9.png)

Does OLM here mean something different?

Reply:
I got from standup that apiserver files are mounted when no annotated service is used but when webhooks configurations are described in OLM manifests.

Reply:
The webhooks are not added to the CSV yet, they're still mounted via secret. I think a follow-up task is to check how adding the webhooks to the CSV will work (if e.g. it'll resolve our webhook delay to have them created by OLM) but it's not done yet to keep the scope of this PR smaller.

Potential issues we'll run into are
- We check/modify webhooks in the controller itself, which could potentially lead to a loop if the specs go out of sync
- We need to manage two separate flows (webhooks created by controller vs webhooks created by OLM)
- We'll need to figure out how CSV generation works to ensure adding webhooks to the CSV isn't a manual step.